scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -0,0 +1,72 @@
+import * as XLSX from "xlsx";
+import { RxNormDrugOption } from "./models";
+import { parseToHOCON, writeFile } from "./utils/common";
+import * as inquirer from "inquirer";
+
+const inquirerFileTreeSelection = require("inquirer-file-tree-selection-prompt");
+
+interface InputOutput {
+  files: string;
+  outputName: string;
+}
+let inputAndOutput: InputOutput;
+
+inquirer.registerPrompt("file-tree-selection", inquirerFileTreeSelection);
+
+inquirer
+  .prompt([
+    {
+      root: "./",
+      type: "file-tree-selection",
+      name: "files",
+      message: "choose RxNorm .txt file"
+    },
+    {
+      type: "input",
+      name: "outputName",
+      message: "Enter output file name. File will be generated on TXT extension"
+    }
+  ])
+  .then((answers: InputOutput) => {
```

Comment:
This is going to be difficult to incorporate into any sort of non-interactive environment- We should not be prompting for input. `npm run` allows for command line arguments to be passed using the `npm run <command> [-- <args>]` format, where `<args>` is one or more key-value pairs. Any kv pairs provided in `args` will be exposed inside `process.env`. For example, `--foo=bar` will be exposed via `process.env.npm_config_foo` and be set to the value `bar`.

Let me know if you have any questions.

Reply:
Added arguments from console and defaults